libs/server/repository/core/application-services/src/lib/read-models/factories/list-repositories-read-model.factory.ts

```diff
@@ -0,0 +1,17 @@
+import { ListRepositoriesReadModel } from '../../queries/list-repositories/list-repositories.read-model';
+import { RepositoryEntity } from '@pimp-my-pr/server/repository/core/domain';
+
+export const listRepositoriesReadModelFactory = (
```

Comment:
Maybe just a constructor?

Reply:
I could not find a reason to use a constructor here.